First/src/firstPackage/EqualOccurances.java

```diff
@@ -0,0 +1,39 @@
+package firstPackage.First.src.firstPackage;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class EqualOccurances {
+    public static boolean NumberOfOccurances(String input) {
```

Comment:
change function name to numberOfOccurences

Reply:
Done!